RocketChat/Rocket.Chat.ReactNative
# Regression: "Join call" button not showing

## Proposed changes

Regression: "Join call" button not showing

## Issue(s)	


## How to test or reproduce

- Start a Video/Voice call
- when back to room, the button `Click to join` should appear

## Screenshots

## Types of changes




## Checklist



## Further comments
